fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/DefaultOCFLObjectSessionFactory.java

```diff
@@ -52,39 +56,36 @@
     @Value("${fcrepo.autoversioning.enabled:true}")
     private boolean autoVersioningEnabled;
 
-    private File ocflStagingDir;
+    private final Path ocflStagingDir;
 
     @Inject
     private MutableOcflRepository ocflRepository;
 
-    /**
-     * Default Constructor.  You can set the ocfl staging, storage root, and work directories by setting the following
-     * system properties: fcrepo.ocfl.staging.dir, fcrepo.ocfl.storage.root.dir, and  fcrepo.ocfl.work.dir.  If these
-     * are not set, default directories will be created in java.io.tmpdir.
-     */
-    public DefaultOCFLObjectSessionFactory() {
-        this(new OCFLConstants().getStagingDir());
-    }
-
     /**
      * Constructor
      *
      * @param ocflStagingDir     The OCFL staging directory
      */
-    public DefaultOCFLObjectSessionFactory(final File ocflStagingDir) {
+    @Autowired
+    public DefaultOCFLObjectSessionFactory(
+            @Value("#{ocflPropsConfig.fedoraOcflStaging}")
+            final Path ocflStagingDir) {
         LOGGER.info("Fedora OCFL persistence staging directory:\n- {}",
```

Comment:
Maybe reduce this logging to `debug`, in favor of the cleaner output of `OcflPropsConfig`.

Reply:
done